### PR TITLE
Adding export-settings api

### PR DESCRIPTION
**Pull request by labhvam5**

### CODE REVIEW

File: apps/fyle/models.py [hunk 1]
```diff
@@ -38,6 +38,15 @@
     'PERSONAL_CORPORATE_CREDIT_CARD_ACCOUNT': 'CCC'
 }
 
+EXPENSE_STATE = {
```
Comment (ashwin1111) [inline apps/fyle/models.py:41]: `DEFAULT_EXENSE_IMPORT_STATE`

File: apps/fyle/models.py [hunk 2]
```diff
@@ -38,6 +38,15 @@
...
+    'REIMBURSABLE': 'PAYMENT_PROCESSING',
+    'CCC': 'PAID'
+}
+
+EXPENSE_GROUP_FIELDS = {
```
Comment (ashwin1111) [inline apps/fyle/models.py:46]: `DEFAULT_EXPENSE_GROUP_FIELDS`

File: apps/fyle/models.py [hunk 3]
```diff
@@ -158,36 +167,37 @@ def create_expense_objects(expenses: List[Dict], workspace_id: int):
 
         return expense_objects
 
-
 def get_default_expense_group_fields():
```
Comment (ashwin1111) [inline apps/fyle/models.py:170]: Why is this function needed? I see default in the model is reading from the dict directly

File: apps/fyle/models.py [hunk 4]
```diff
@@ -158,36 +167,37 @@ def create_expense_objects(expenses: List[Dict], workspace_id: int):
...
-    return ['employee_email', 'report_id', 'claim_number', 'fund_source']
-
+    return EXPENSE_GROUP_FIELDS['REIMBURSABLE']
 
 def get_default_expense_state():
```
Comment (ashwin1111) [inline apps/fyle/models.py:173]: Why is this function needed? I see default in the model is reading from the dict directly

File: apps/fyle/models.py [hunk 5]
```diff
@@ -38,6 +38,15 @@
     'PERSONAL_CORPORATE_CREDIT_CARD_ACCOUNT': 'CCC'
 }
 
+EXPENSE_STATE = {
+    'REIMBURSABLE': 'PAYMENT_PROCESSING',
+    'CCC': 'PAID'
```
Comment (ashwin1111) [inline apps/fyle/models.py:43]: both the values should be `PAYMENT_PROCESSING` only

File: apps/workspaces/models.py [hunk 6]
```diff
@@ -8,8 +8,27 @@
...
+    ('IMPORT_SETTINGS', 'IMPORT_SETTINGS'),
+    ('ADVANCED_CONFIGURATION', 'ADVANCED_CONFIGURATION'),
+    ('COMPLETE', 'COMPLETE')
+)
+
+#add or remove this after discussion.
+AUTO_MAP_EMPLOYEE = (
+    ('EMAIL', 'EMAIL'),
+    ('NAME', 'NAME'),
+    ('EMPLOYEE_CODE', 'EMPLOYEE_CODE')
```
Comment (ashwin1111) [inline apps/workspaces/models.py:23]: This option is removed

File: apps/workspaces/models.py [hunk 7]
```diff
@@ -8,8 +8,27 @@
...
+)
+
+
 def get_default_chart_of_accounts():
     return ['EXPENSE']
+
+def get_default_onboarding_state():
+    return 'CONNECTION'
```
Comment (ashwin1111) [inline apps/workspaces/models.py:31]: new line after func

File: apps/fyle/models.py (hunk 1 above)
Comment (labhvam5) [reply apps/fyle/models.py:41]: Have removed this and moved the default to hardcoding it the while declaring the filed.

File: apps/fyle/models.py (hunk 2 above)
Comment (labhvam5) [reply apps/fyle/models.py:46]: have removed this and move it to a function based approach .

File: apps/fyle/models.py (hunk 5 above)
Comment (labhvam5) [reply apps/fyle/models.py:43]: Done

File: apps/workspaces/models.py (hunk 6 above)
Comment (labhvam5) [reply apps/workspaces/models.py:23]: Will have to do it at the end have created a click-up for this.

File: apps/workspaces/models.py (hunk 7 above)
Comment (labhvam5) [reply apps/workspaces/models.py:31]: done

File: apps/fyle/models.py (hunk 3 above)
Comment (labhvam5) [reply apps/fyle/models.py:170]: Have removed the dict and added this back as django was throwing some error.

File: apps/fyle/models.py (hunk 4 above)
Comment (labhvam5) [reply apps/fyle/models.py:173]: Have removed the dict and added this back as django was throwing some error.

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**